confluence-slack-server-integration-plugin/src/main/java/com/atlassian/confluence/plugins/slack/spacetochannel/configuration/SpaceToChannelConfiguration.java

```diff
@@ -117,7 +118,7 @@ public SpaceToChannelConfiguration build() {
          * @param channelId the channel id the SpaceToChannelSettings.Builder is for
          * @return a SpaceToChannelSettings.Builder
          */
-        public SpaceToChannelSettings.Builder getSettingsBuilder(String channelId) {
+        public SpaceToChannelSettings.Builder getSettingsBuilder(ConversationKey channelId) {
```

Comment:
Please rename `channelId` to `conversationKey` in order to make it easier to detect variable type by its name. It may be done in another PR.